### PR TITLE
add comment-form block styles

### PR DESCRIPTION
**Pull request by aristath**
## Description

The post-comment-form block depends on the block button's styles. After #25220 got merged, FSE themes will not load the button block's styles unless a button block exists on the page.
This PR adds the button block's styles to the comment form.

Note: This is a less-than-ideal solution. Ideally we'd be able to have style dependencies and the ability to load multiple stylesheets as outlined in #27671.

## Screenshots 

Before:

Screenshot_2020-12-11 Hello world – Gutenberg

After:

Screenshot_2020-12-11 Block Button – Gutenberg

## Types of changes

Defined `wp-block-button` as the stylesheet handle for the `post-comment-form` block.

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [x] My code has proper inline documentation. 
- [x] I've included developer documentation if appropriate. 
- [x] I've updated all React Native files affected by any refactorings/renamings in this PR.

### CODE REVIEW

**Comment by gziolo** [comment]
I would rather say that it's an issue with the block that it assumes that CSS from another block exists on the page.

**Comment by aristath** [comment]
> I would rather say that it's an issue with the block that it assumes that CSS from another block exists on the page.

I agree. But at the time it made sense since everything was in the same stylesheet...
This is mostly a quick-n-dirty fix until we figure out how to handle situations like these. That's why I created #27671 where we can discuss possible solutions to this and other issues that came up 👍

**Comment by gziolo** [comment]
Is is a lot of styles to copy and paste? I personally would be in favor of doing that and scoping those styles to this block.

**Comment by aristath** [comment]
Fair point... Done in https://github.com/WordPress/gutenberg/pull/27673/commits/a4a41d5c07393b3a99b0ff20cd106175e3c1f6ac :+1:

**Review by gziolo — COMMENTED** [review]
I think it should be also listed as import in the main SCSS file in the block library package. Well, at this moment maybe it's unnecessary but it might change one day 😅

**Review by gziolo — APPROVED** [review]

**Comment by gziolo** [comment]
To clarify my previous comment, I was referring to this file:

https://github.com/WordPress/gutenberg/blob/master/packages/block-library/src/style.scss

**Comment by aristath** [comment]
> I think it should be also listed as import in the main SCSS file in the block library package. Well, at this moment maybe it's unnecessary but it might change one day sweat_smile

Good catch! Done :+1:

File: packages/block-library/src/post-comments-form/style.scss [hunk 1]
```diff
@@ -0,0 +1,24 @@
+$blocks-button__height: 3.1em;
+
+// Styles copied from button block styles.
+.wp-block-post-comments-form input[type="submit"] {
+	color: $white;
+	background-color: #32373c;
```
Comment (gziolo) [inline packages/block-library/src/post-comments-form/style.scss:6]: Is there any variable that matches this color?

**Review by gziolo — APPROVED** [review]
Thank you for revisions added 💯

File: packages/block-library/src/post-comments-form/style.scss (hunk 1 above)
Comment (aristath) [reply packages/block-library/src/post-comments-form/style.scss:6]: Nope! It's the exact same CSS used in the block button... I grep'd the repo and there's no variable defined for that color (css-var or scss-var)

**Pull request merged**